engine/flink/management/sample/src/main/scala/pl/touk/nussknacker/engine/management/sample/dict/BusinessConfigDictionary.scala

```diff
@@ -0,0 +1,19 @@
+package pl.touk.nussknacker.engine.management.sample.dict
+
+import pl.touk.nussknacker.engine.api.dict.DictInstance
+import pl.touk.nussknacker.engine.api.dict.embedded.EmbeddedDictDefinition
+
+object BusinessConfigDictionary {
```

Comment:
could we add some comment what feature it shows? At least for new components? After a while we forget about that :)